### PR TITLE
bump to Kubernetes 1.19

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,16 +7,16 @@ require (
...
-	k8s.io/apiextensions-apiserver v0.18.0-beta.2
-	k8s.io/apimachinery v0.18.0-beta.2
-	k8s.io/client-go v0.18.0-beta.2
-	k8s.io/code-generator v0.18.0-beta.2
+	k8s.io/api v0.19.0
+	k8s.io/apiextensions-apiserver v0.19.0
+	k8s.io/apimachinery v0.19.0
+	k8s.io/client-go v0.19.0
+	k8s.io/code-generator v0.19.0
 	k8s.io/klog v1.0.0
```
Comment (sanchezl) [inline go.mod:17]: Should we migrate to klog/v2 to be in sync with kube 1.19.x ?

Comment (caesarxuchao) [reply go.mod:17]: Good point. We shouldn't mix klog/v1 and v2.

I'd suggest creating one PR to upgrade to klog/v2, and then another one to upgrade to k8s 1.19.

File: go.mod [hunk 2]
```diff
@@ -3,20 +3,18 @@ module sigs.k8s.io/kube-storage-version-migrator
 go 1.13
 
 require (
-	github.com/google/go-cmp v0.3.1 // indirect
-	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/onsi/ginkgo v1.11.0
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.7.1
```
Comment (caesarxuchao) [inline go.mod:8]: Why do we change prometheus and spf13 as well? Are they dependencies of k8s 1.19?

**Comment by caesarxuchao** [comment]
/close

We are using kubernetes 1.20 now.

**Pull request closed without merging**